cmd/cosign/cli/verify.go

```diff
@@ -146,6 +146,12 @@ func (c *VerifyCommand) Exec(ctx context.Context, args []string) (err error) {
 	co.SigVerifier = pubKey
 
 	for _, imageRef := range args {
+
+		if imageRef == "scratch" {
```

Comment:
Should we normalize the case here on imageRef to support SCRATCH? or is that handled somewhere else too

Reply:
I think it's unnecessary. A test with a `FROM SCRATCH` Dockerfile shows repositories must be lowercase.

```
failed to solve with frontend dockerfile.v0: failed to create LLB definition: failed to parse stage name "SCRATCH": invalid reference format: repository name must be lowercase 
```